docs/content/doc/advanced/config-cheat-sheet.en-us.md

```diff
@@ -82,7 +82,7 @@ Values containing `#` or `;` must be quoted using `` ` `` or `"""`.
 
 ## Server (`server`)
 
-- `PROTOCOL`: **http**: \[http, https, fcgi, unix\]
+- `PROTOCOL`: **http**: \[http, https, fcgi, unix, letsencrypt\]
```

Comment:
Then if you use `letsencrypt` you should set up the DOMAIN correctly.

Reply:
updated